app/ts/Workspace/transformUtil.test.ts

```diff
@@ -0,0 +1,67 @@
+import ICriterion from '@shared/interface/ICriterion';
+import IDataSource from '@shared/interface/IDataSource';
+import IProblemCriterion from '@shared/interface/Problem/IProblemCriterion';
+import IProblemDataSource from '@shared/interface/Problem/IProblemDataSource';
+import {
+  transformCriterionToOldCriterion,
+  transformDataSourcesToOldDataSources
+} from './transformUtil';
+
+describe('transformUtil', () => {
+  describe('transformCriterionToOldCriterion', () => {
+    it('should rename isFavourable into isFavorable (data source transform in seperate test)', () => {
```

Comment:
oh noooooo we're mixing spellings again?

Reply:
Yes :( But the 'o' variant should be all again once angular is out.